docs/docs/getting-started/using-files.md

```diff
@@ -50,40 +40,45 @@ To use absolute paths in Lug, you'll need:
 ```python
 import lug
 import os
-import subprocess
 
-@lug.run(image="alpine:3.16.2", mount="/")
-def hello_world(file_path):
-    result = subprocess.run(f"cat /lug{file_path}", text=True, capture_output=True, shell=True)
+@lug.docker_sidecar(sidecar_image="alpine:3.16.2", mount="/")
+def cat_file(absolute_path):
+    result = lug.sidecar_shell(f"cat /lug{absolute_path}")
     return result.stdout
 
-
-file_path = os.path.abspath("test.txt")
-with open(file_path, 'w') as f:
+absolute_path = os.path.abspath("test.txt")
+with open(absolute_path, 'w') as f:
     f.write('Hello, world!')
 
-print(hello_world(file_path))
+print(cat_file(absolute_path))
 ```
 
-You'll still see `Hello, world!` printed.
+If everything is working, you'll see `Hello, world!` printed.
 
 ## Using files remotely
 
-With remote execution, there are two new ways to access files:
+To use remote files, you'll need to set up input and output file handling.
+
+### Input files
+
+All input files are passed to the Lug` remote_inputs` argument. They're accessible at `./input/` on the remote instance.
+
+
+### Output files
 
-1. You need to pass all input files to the Lug` remote_inputs` argument. They're accessible at `./input/`.
+All output files need to be written to `./output` on the remote instance. Set the `remote_output_directory` argument to 
+a directory where Lug will transfer all the output files.
```

Comment:
"Set the `remote_output_directory` argument to 
a directory where Lug will transfer all the output files." I feel like there's a clearer where to say that this  a local directory where remote output will be downloaded. Also maybe rename the arg in the longer term.

Reply:
Possibly fixed, let me know what you think about the new wording